Document required lombok.config value (#221)
kelloggm/checkerframework-gradle-plugin: README.md

to delombok your source code before it is passed to the Checker Framework
for typechecking. This plugin does not support any other use of Lombok.

> For the Checker Framework to work properly on code that is generated by Lombok,

Reviewer: I think this may be clearer?  Feel free to tweak further
```suggestion
For the Checker Framework to work properly on delombok'd source code,
```
